add s3 known limitations to sigv4 (#11811)
DataDog/documentation: content/en/synthetics/api_tests/http_tests.md

   * **HTTP Basic Auth**: Add HTTP basic authentication credentials.
   * **AWS Signature V4**: Enter your Access Key ID and Secret Access Key. Datadog generates the signature for your request.

>    This option uses the basic implementation of SigV4.
>    Known limitations: AWS S3 specific signature is not implemented.

Reviewer: ```suggestion
   This option uses the basic implementation of SigV4. AWS S3-specific signatures are not implemented.
```

>    This option uses the basic implementation of SigV4.
>    Known limitations: AWS S3 specific signature is not implemented.

Reviewer 2: ```suggestion
   This option uses the basic implementation of SigV4. The AWS S3 specific signature is not implemented.
```
